examples/artemis-transaction-producer/artemis_transaction_producer.bal

```diff
@@ -0,0 +1,23 @@
+import ballerina/artemis;
+import ballerina/io;
+
+public function main() {
+    // Create the Artemis `Producer` with `autoCommitSends` set to `false`.
+    // In this case the message sends will be committed only
```

Comment:
```suggestion
    // In this case, the message will be sent only
```

Reply:
It is that we are committing sends. This is for transactions.